geanyvc/src/geanyvc.c

```diff
@@ -2293,7 +2294,8 @@ plugin_configure(GtkDialog * dialog)
 	if (lang != NULL)
 		gtk_entry_set_text(GTK_ENTRY(widgets.spellcheck_lang_textbox), lang);
 
-	gtk_misc_set_alignment(GTK_MISC(label_spellcheck_lang), 0, 0.5);
+	gtk_widget_set_halign(GTK_WIDGET(label_spellcheck_lang), GTK_ALIGN_START);
+	gtk_widget_set_valign(GTK_WIDGET(label_spellcheck_lang), GTK_ALIGN_CENTER);
```

Comment:
I'd leave the default `valign` of `FILL` (i.e. not explicitly set it: what this was after is left-align the label)

```suggestion
```